docker-compose.yml

```
@@ -0,0 +1,62 @@
+services:
+  db:
+    image: mariadb
+    restart: always
+    environment:
+      MARIADB_ROOT_PASSWORD: myfinrootpassword
+      MARIADB_DATABASE: myfin
+      MARIADB_USER: myfin
+      MARIADB_PASSWORD: myfinpassword
+    volumes:
+      - db_data:/var/lib/mysql
+    healthcheck:
+      test: [ "CMD", "healthcheck.sh", "--connect", "--innodb_initialized" ]
+      start_period: 10s
+      interval: 10s
+      timeout: 5s
+      retries: 3
+
+  myfin-api:
+    image: ghcr.io/afaneca/myfin-api:latest
+    container_name: myfin-api
+    restart: unless-stopped
+    ports:
+      - 8081:3001
```

```suggestion
      - "8081:3001"
```
Unquoted port mapping not recommended